WordPress/Classes/Services/LikeUserHelpers.swift

```diff
@@ -58,14 +86,8 @@ import CoreData
         preferredBlog.user = user
     }
 
-    class func purgeStaleLikes() {
-        ContextManager.shared.performAndSave {
-            purgeStaleLikes(fromContext: $0)
-        }
-    }
-
     // Delete all LikeUsers that were last fetched at least 7 days ago.
-    private class func purgeStaleLikes(fromContext context: NSManagedObjectContext) {
+    class func purgeStaleLikes(fromContext context: NSManagedObjectContext) {
         guard let staleDate = Calendar.current.date(byAdding: .day, value: -7, to: Date()) else {
```

Comment:
Nitpick / Out of context, but jumped to my eye.

Injecting the `Date()` parameter would make tests more straightforward because we could use static dates instead of computing them all the time.

```suggestion
    class func purgeStaleLikes(fromContext context: NSManagedObjectContext, referenceDate: Date = Date()) {
        guard let staleDate = Calendar.current.date(byAdding: .day, value: -7, to: Date()) else {
```

One could make the method even more flexible by injecting the `.day` and `-7` parameters, but it might be overengineering at this point.